menu.js

```diff
@@ -0,0 +1,12 @@
+const hamburgerBtn = document.getElementById('navicon');
+const menuIcon = document.getElementById('menu');
+const btnCross = document.getElementById('cross');
+
+// navLink.style.left = "-100%";
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They will make your project look messy. If you need to review a previous version of your code, you can always use git history. 🤝